torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12000,6 +12022,30 @@ def ref_pairwise_distance(input1, input2):
            ),
            sample_inputs_func=sample_inputs_atleast1d2d3d,
            ),
+    OpInfo('flatten',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_out=False,
+           supports_forward_ad=True,
+           sample_inputs_func=sample_inputs_flatten,
+           skips=(
+               # File "pytorch/torch/testing/_internal/common_jit.py", line 306, in test_type
+               # self.assertTrue(actual_type.isSubtypeOf(out_type))
+               # AssertionError: False is not true
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
+           )
+           ),
+    OpInfo('column_stack',
+           dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16),
+           supports_forward_ad=True,
+           skips=(
+               # JIT does not support variadic tensors.
+               # RuntimeError: input->type()->kind() == TypeKind::OptionalType
+               # INTERNAL ASSERT FAILED at "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":252,
+               # please report a bug to PyTorch.
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit', dtypes=(torch.float32,)),
+               # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+               DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_out', dtypes=(torch.float32,)),),
```

Comment:
Can this be xfailed?

Reply:
Again seems like something we should fix and is not expected

Reply:
Again I think this should be fixed and is not expected